apps/fabric-website-resources/webpack.config.js

```diff
@@ -26,6 +26,7 @@ module.exports = [
       alias: {
         'office-ui-fabric-react/src': path.resolve(__dirname, '../../packages/office-ui-fabric-react/src'),
         'office-ui-fabric-react/lib': path.resolve(__dirname, '../../packages/office-ui-fabric-react/lib'),
+        '@uifabric/api-docs/lib': path.join(__dirname, 'node_modules/@uifabric/api-docs/lib'),
```

Comment:
Should this be reverted prior to merge with `master`?

Reply:
This seemed to be needed to help webpack resolve the package name.